lib/helpers.js

```diff
@@ -0,0 +1,43 @@
+// Figure out if the value is a number or a string containing only numbers
+exports.isNumber = (value) => {
+    /* istanbul ignore next */
+    if (typeof value === 'number') {
+        return true;
+    }
+
+    /* istanbul ignore else */
+    if (typeof value === 'string') {
+        const valueAsNumber = +value; // converts to number if it's all numbers or to NaN otherwise
+        return !Number.isNaN(valueAsNumber);
+    }
+
+    /* istanbul ignore next */
+    return false;
+};
+
+// A helper to determine if user has permission.
+function hasPermission(permissionsList, combinedPermission) {
+    return permissionsList.some((permission) => permission.combined.endsWith(combinedPermission));
+}
+
+exports.getPermissions = (user, corePermissions) => {
+    return {
+        boards: hasPermission(corePermissions, 'manage:boards')
```

Comment:
it's not clear what it's doing from the name, can we add something containing "network" here? either that or comment what does it do

Reply:
What do you mean? It retrieves a certain permission from core, but now that I'm looking at it I'm not sure if I use it in #2 so I will look at it again

Reply:
I mean that from "global:manage:boards" it's not clear which service is using it